.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -12,7 +12,6 @@ jobs:
         CONFIG: linux_64_
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
-  swapfile_size: 0GiB
```

Comment:
Note that the problematic line mentioned in issue ( https://github.com/conda-forge/conda-smithy/issues/1784 ) is being removed

cc @BEFH (for awareness)